src/resolve.c

```diff
@@ -386,7 +386,7 @@ static void resolveClients(const bool onlynew)
 
 		// Only try to resolve host names of clients which were recently active if we are re-resolving
 		// Limit for a "recently active" client is two hours ago
-		if(onlynew == false && client->lastQuery < now - 2*60*60)
+		if(!force_refreshing && onlynew == false && client->lastQuery < now - 2*60*60)
```

Comment:
```suggestion
		if(!force_refreshing && !onlynew && client->lastQuery < now - 2*60*60)
```

Just for consistency, or

```suggestion
		if(force_refreshing == false && onlynew == false && client->lastQuery < now - 2*60*60)
```